tests/unittests/tests-hash_string/tests-hash_string.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2015 Kushal Singh <kushal.spiderman.singh@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include "tests-hash_string.h"
+#include "hash_string.h"
+
+static void test_hash_value(void)
+{
+    TEST_ASSERT_EQUAL_INT(177621, hash_string("0"));
+    TEST_ASSERT_EQUAL_INT(6555485335635392041, hash_string("0123456789abcde-0123456789abcde-0123456789abcde-0123456789abcde-"));
```

Comment:
Can you add a linebreak somewhere? You can concat string constants like that in C:
```c
char *str = "abcd" "efgh"; /* no operator in between */
```